src/Symfony/Component/Notifier/Bridge/AmazonSns/AmazonSnsOptions.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\AmazonSns;
+
+use AsyncAws\Sns\Input\PublishInput;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Adrien Chinour <github@chinour.fr>
+ */
+final class AmazonSnsOptions implements MessageOptionsInterface
+{
+    private $options = [];
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public function toArray(): array
+    {
+        $options = $this->options;
+        unset($options['recipient_id']);
```

Comment:
Why storing the recipient in the `option` property if we don't use it not a standard option?

Reply:
@jderusse Yes, I change it to a class property and a required parameter on constructor.